inc/taihostif.h

```diff
@@ -87,15 +87,15 @@ typedef enum _tai_host_interface_attr_t
      *
      * A list of lane fault status
      *
-     * @type #tai_u32_list_t #tai_host_interface_lane_faults_t
+     * @type #tai_s32_list_t #tai_host_interface_lane_fault_t
      * @flags READ_ONLY
      */
-    TAI_HOST_INTERFACE_ATTR_LANE_FAULTS,
```

Comment:
Since this is a list, the plural "FAULTS" seems appropriate. What was the reason for this change?

Reply:
Same reason as above.

`SAI_PORT_ATTR_SUPPORTED_FEC_MODE` is an enum list but it is not `SAI_PORT_ATTR_SUPPORTED_FEC_MODES`

I think it's clearer to make the attr name similar to the attr type name which is `tai_host_interface_lane_fault_t` in this case.

Reply:
Ok, I'm fine with this. At some point we need to be a little more restrictive when changing the names of attributes. Every time we do this, all of the code (tai adapters and tai applications) must be modified to use the new names.

Reply:
Yes, I agree. At this point, I don't have any more proposal for the naming change.